packages/shared/src/features/onboarding/types/funnel.ts

```diff
@@ -45,8 +45,16 @@ export const COMPLETED_STEP_ID = 'finish' as const;
 export type FunnelStepTransitionCallback<Details = Record<string, unknown>> =
   (transition: { type: FunnelStepTransitionType; details?: Details }) => void;
 
-interface FunnelStepCommonParameters {
+interface FunnelBannerMessage {
+  image: {
+    src: string;
+  };
```

Comment:
I'm kinda torn in keeping it just a string or following this new convention. We're already having dynamic images on other areas and they don't follow this convention yet.

Reply:
I'm cool with any convention, just wanted if we can be more consistent moving forward and if we should follow this new one.